tools/poctools/requirements.txt

```diff
@@ -23,4 +24,3 @@ tqdm==4.48.2
 twine==3.2.0
 urllib3==1.25.10
 webencodings==0.5.1
-wheel==0.35.1
```

Comment:
Not sure why this is removed, but I don't have wheel installed either in the virtual environment nor globally, so I guess it's not needed.